src/_pytest/python.py

```diff
@@ -568,8 +568,7 @@ def __init__(
         nodes.FSCollector.__init__(
             self, fspath, parent=parent, config=config, session=session, nodeid=nodeid
         )
-
-        self.name = fspath.dirname
+        self.name = os.path.basename(str(fspath.dirname))
```

Comment:
that one looks like a breaking change in isolation, does it bring us in line with other details or is it a new difference?

Reply:
crosschecked, it brings us in line